### ISSUE TITLE
New preset: amenity=music_venue

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7605]
A place for concerts.
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dmusic_venue

**Comment by Adamant36** [comment]
There doesn't seem to be a good consensus on what tag to use for music venues unfortunately. As reflected by the low usage and the proposal never going anywhere.

**Comment by maro-21** [comment]
Do you have any other alternative?

**Comment by Adamant36** [comment]
> Do you have any other alternative?

Nope. Like I said, there isn't currently a good tag for "music venues" and I'm not really into inventing random ones.
